packages/manager/modules/pci/src/projects/project/ai/apps/dashboard/general-information/update-scaling/update-scaling.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.ai.apps.dashboard.general-information.update-scaling',
+    {
+      url: '/update-scaling',
+      // params: { app: null },
+      views: {
+        modal: {
+          component: 'ovhManagerPciProjectAppsUpdateAppScaling',
+        },
+      },
+      layout: 'modal',
+      resolve: {
+        // app: /* @ngInject */ ($transition$) => $transition$.params().app,
```

Comment:
**nitpick:** Can you remove the comment if `app` is not required?

Reply:
Sure ! https://github.com/ovh/manager/pull/8001/commits/33a7214d2b0990fa80eb54768cc5b204824233b1